pkg/export/setup/setup.go

```diff
@@ -96,6 +96,9 @@ func FromFlags(a *kingpin.Application, userAgent string) func(log.Logger, promet
 	a.Flag("export.endpoint", "GCM API endpoint to send metric data to.").
 		Default("monitoring.googleapis.com:443").StringVar(&opts.Endpoint)
 
+	a.Flag("export.compression", "The compression format to use for gRPC requests ('none' or 'gzip').").
```

Comment:
Yea I doubt it will be a user-visible issue on GCP. If it becomes an issue for other users I'd rather consider just enabling it by default – the CPU usage certainly increases but probably not to a degree at which we'd need to expose controls.